index.js

```diff
@@ -36,12 +36,11 @@ module.exports = class WebpackCopyModulesPlugin {
   }
 
   handleEmit(compilation) {
-    return Promise.all(compilation.modules.map(this.saveModule.bind(this)));
-  }
-
-  saveModule(module) {
     const me = this,
-        fileDependencies = module.buildInfo.fileDependencies || new Set();
+          fileDependencies = new Set();
```

Comment:
Minor: we stick to double indentation for this case, even though with length of the word "const" makes triple indentation a reasonable choice as well